src/encoded/tests/test_batch_download.py

```diff
@@ -478,9 +449,9 @@ def test_metadata_tsv_fields(testapp, workbook):
         _tsv_mapping,
         _excluded_columns,
     )
-    r = testapp.get('/metadata/type%3DExperiment/metadata.tsv')
-    metadata_file = r.body.decode('UTF-8').split('\n')
-    actual_headers = metadata_file[0].split('\t')
-    assert len(actual_headers) == len(set(actual_headers))
+    r = testapp.get('/metadata/?type=Experiment')
```

Comment:
are you changing the URL here?   There's no longer a metadata.tsv at the end?   We will have to forward them somehow.  People can have old files.txt

Reply:
Yes! The URL is changing. Moving from params as part of the path to query strings is by definition changing both URLs. If we keep a redirect that accepts search params we will run into the same double unquote URL file type bug that we are trying to avoid by moving to query strings. I removed the metdata.tsv ending because it's not consistent with batch_download (which also returns a file but doesn't have a batchdownload.txt/tsv ending).

Reply:
So we will have to rewrite the URL at the apache level for backwards compatibility.

Reply:
Added.